homeassistant/components/modbus/base_platform.py

```diff
@@ -197,6 +230,18 @@ def unpack_structure_result(self, registers: list[int]) -> str | None:
 
         # Apply scale and precision to floats and ints
         val_result: float | int = self._scale * val[0] + self._offset
+        if (
+            self._min_value_threshold is not None
```

Comment:
please remove min_value_threshold

Reply:
min value on its own sets a value it cannot be smaller than BUT ...

This will not work for my use case ... I would have to add another setting

i.e. how do you suggest we achieve: if it is less than 0.025 mA, treat it as 0

I am open to another suggestion, as min_value on its own will not.